Makefile

```
@@ -1,5 +1,5 @@
-#CUDA_HOME=/usr/local/cuda
-CUDA_HOME=/usr/local/cuda-8.0
+CUDA_HOME=/usr/local/cuda
+#CUDA_HOME=/usr/local/cuda-8.0
```

```suggestion
CUDA_HOME=$(CUDADIR)
```
where `CUDADIR` is an environment variable (you choose the name! - I've looked on the Lancaster GPU machine and can't see any default variable)
Then add to the `README.md` file that you need to set that variable in order to use the GPU code